setup.py

```diff
@@ -112,6 +112,8 @@ def read(path, encoding="utf-8"):
 
 chromadb_libs = ["chromadb"]
 
+milvus_libs = ["pymilvus>=2.3.0"]
```

Comment:
Are there any breaking changes in the previous versions?

Reply:
Only Milvis >= 2.3.0 has native support for cosine similarity, which the `MilvusVectorStore` uses for searching. I chose cosine similarity since the other vector index integration uses it. If we were to switch to IP or L2, then earlier versions can be used.